### PR TITLE
Update troubleshooting docs for Cloud

### PR DESCRIPTION
**Pull request by AlexanderVK-SLC**

### CODE REVIEW

File: user-guide/Cloud_Platform/RemoteAccess/Accessing_DMS_remotely_with_Cube.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ From DataMiner 10.3.0/10.3.2 onwards, you can use DataMiner Cube to access your
 - You have been [granted access to dataminer.services features](xref:Giving_users_access_to_cloud_features).
 - You [have the DataMiner Cube desktop application installed](xref:Installing_configuring_the_DataMiner_Cube_software).
 - Your user account has been granted access to the DataMiner System.
+- You have the [APIGateway](xref:DataMinerExtensionModules#apigateway) installed and running.
```
Comment (MariekeGO) [inline user-guide/Cloud_Platform/RemoteAccess/Accessing_DMS_remotely_with_Cube.md:18]: The way this is phrased now, it looks like APIGateway has to be installed on the client machine. Is this indeed what you mean, or do you mean that it must be installed on the DMA the user is trying to connect to?

Comment (AlexanderVK-SLC) [reply user-guide/Cloud_Platform/RemoteAccess/Accessing_DMS_remotely_with_Cube.md:18]: It is indeed required to be on the system not the client.

File: user-guide/Troubleshooting/Procedures/Cloud_Connection_Issues.md [hunk 2]
```diff
@@ -56,6 +53,20 @@ For each DMA that should be connected, verify whether the DMA is operational and
...
+
+If the connection status is shown as red on [dataminer.services](https://dataminer.services/), validate that at least one CloudGateway instance is running in the cluster and that it can establish a connection towards dataminer.services:
+
+1. On the DMA servers, go to the Windows *Services* configuration window.
+
+1. Check whether the *DataMiner CloudGateway* service has the status *Running*.
+
+1. If the status is not *Running*, right-click the service and select *Start*.
+
+If there is a CloudGateway instance running, but the dataminer.services connection is still not valid, validate whether your network complies with the requirements for connecting to dataminer.services using the *ConnectionTester.exe* tool from the folder `Program files\Skyline Communications\Dataminer CloudGateway\`. More information on the requirements can be found
```
Comment (MariekeGO) [inline user-guide/Troubleshooting/Procedures/Cloud_Connection_Issues.md:66]: Something is still missing here. Could you add the missing info?

Comment (AlexanderVK-SLC) [reply user-guide/Troubleshooting/Procedures/Cloud_Connection_Issues.md]: Done

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**